src/components/action/validation/validationActionComponents/ValidationSettings.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+
+import CheckBox from '../../checkBox/CheckBox';
+import './ValidationSettings.css';
+import ValidationProfile from './ValidationProfile';
+import FixMetadata from './FixMetadata';
+
+class ValidationSettings extends Component {
+    fileInput = null;
+
+    render() {
+        const titleInformation =
+            'include information on passed rules into the report';
+
+        const {
+            profilesValues,
+            getValidationProfileValue,
+            isFixMetadata,
+            selectFixMetadata,
+            profilePath,
+            includeInformation,
+            selectIncludeInformation,
+            selectPrefix,
+            stopValidation,
+            amountFails,
+            prefix,
+            selectStopValidation,
+            selectAmoutFails,
+            getProfilePath
+        } = this.props;
+
+        return (
+            <div>
+                <ValidationProfile
+                    profilesValues={profilesValues}
+                    getProfilePath={getProfilePath}
+                    profilePath={profilePath}
+                    getValidationProfileValue={getValidationProfileValue}
+                />
+                <div>
+                    <CheckBox
+                        checked={isFixMetadata}
+                        title="Fix metadata:"
+                        onChange={selectFixMetadata}
+                    />
+                </div>
+                <FixMetadata selectPrefix={selectPrefix} prefix={prefix} />
+                <CheckBox
+                    title={titleInformation}
+                    checked={includeInformation}
+                    onChange={selectIncludeInformation}
+                />
+                <div className="validation-checks-fail-container">
+                    <CheckBox
+                        checked={stopValidation}
+                        title="Stop validation after "
```

Comment:
Can we make title to accept not only strings but also complex elements?

So that final markup was similar to:
`<input type='checkbox'><label>Text 1 <input/> text 2</label>`